pkgs/by-name/ai/airscan/package.nix

```
@@ -0,0 +1,25 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "airscan";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "stapelberg";
+    repo = pname;
+    rev = "v${version}";
```

Will help to avoid branch/tag confusion. Happens from time to time.

```suggestion
    rev = "refs/tags/v${version}";
```